### ISSUE TITLE
Don't scatter on play

### ISSUE THREAD

**Issue by mournguard** [arcaneenergy/godot-multimesh-scatter#6]
I don't think the plugin should scatter on play. With some of the new settings it's extremely slow, and for many cases the user might want it exactly as-is. 

If that behavior is not entirely removed I think it should at least be a setting, being off by default.

**Comment by arcaneenergy** [comment]
Yes, this makes sense. If the user needs to scatter at runtime, they can just call the `scatter()` function. So yes, we can remove this behaviour :)
